testdata/scripts/linkname.txt

```diff
@@ -16,6 +16,16 @@ cmp stderr main.stderr
 module test/main
 
 go 1.16
+
+-- a.go --
+package main
+
+// Call a function which is linknamed to make sure we don't obfuscate the
+// call when the name is blacklisted.
```

Comment:
I'd even make this a bit more precise, like:

```
// Call a function which is linknamed to another symbol.
// What's special here is that we obfuscate this call before the function declaration.
// If we decide not to obfuscate the name in the function declaration,
// we shouldn't obfuscate the name here either.
```

The reason for the "if we decide not to obfuscate" is because that will likely change; see https://github.com/burrowers/garble/issues/309. But the test case is good to have one way or another.